scripts/qa_crowbarsetup.sh

```diff
@@ -5262,6 +5267,47 @@ function onadmin_runlist
     done
 }
 
+function onadmin_external_ceph
+{
+    local ceph_cluster=($@)
+    local node
+    local i=0
+    local ip_alloc
+    local ses_repo="http://$adminip:8091/suse-$suseversion/$arch/repos/SUSE-Enterprise-Storage-$sesversion"
+    for node in ${ceph_cluster[@]}; do
+        set_node_alias $node ceph$i
+        let "i+=1"
+        ssh $node zypper ar -f ${ses_repo}-Pool/ SUSE-Enterprise-Storage-$sesversion-Pool
+        ssh $node zypper ar -f ${ses_repo}-Updates/ SUSE-Enterprise-Storage-$sesversion-Updates
+        ip_alloc=0
+        for tries in $(seq 2); do
+            bash ${SCRIPTS_DIR}/allocate_ip.sh $node storage host
+            if [ $want_separate_ceph_network -eq 1 ]; then
+                bash ${SCRIPTS_DIR}/allocate_ip.sh $node ceph host
+            else
+                bash ${SCRIPTS_DIR}/allocate_ip.sh $node public host
+            fi
+            if [ $? -eq 0 ]; then
+                ip_alloc=1
+                break
+            fi
+        done
+        if [ $ip_alloc -eq 0 ]; then
+            echo "Failed to assign a network to $node"
+            return
+        fi
+    done
+    if [ -z $net_storage ]; then
+        setcloudnetvars $cloud
+    fi
+    echo "== Running SES deployment script =="
+    if [ $want_separate_ceph_network -eq 1 ]; then
+        safely bash ${SCRIPTS_DIR}/build_ses.sh ${net_ceph}.0/24 ${net_storage}.0/24 ${ceph_cluster[@]}
+    else
+        safely bash ${SCRIPTS_DIR}/build_ses.sh ${net_public}.0/24 ${net_storage}.0/24 ${ceph_cluster[@]}
```

Comment:
Why not just make the scripts executable?
... and then use:
`safely ${SCRIPTS_DIR}/build_ses.sh ...`